samples/text_generation.py

```diff
@@ -23,15 +23,19 @@
 class UnitTests(absltest.TestCase):
     def test_text_gen_text_only_prompt(self):
         # [START text_gen_text_only_prompt]
+        import google.generativeai as genai
+        import os
+
+        genai.configure(api_key=os.environ["GOOGLE_API_KEY"])
```

Comment:
If the API key is set as an environment variable the SDK will automatically pick it up. Do we still want this?